index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="href='https://fonts.googleapis.com/css?family=Poppins" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    
+</head>
+
+<body>
+
+
+    <header>
+        <nav class="topnav">
+            <h2 class="logo">My Logo</h2>
+
+            <a class="icon">
+                <i class="fa fa-bars"></i>
+            </a>
+
+        </nav>
+    </header>
```

Comment:
## Toolbar section 🎈🎈
- [ ] I think it would be better to use a ```<a>``` tag on your logo to instead of using the ```<h2>``` This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. 😎😎
- [ ] I think it would be better also to remove the **logo** into the```<nav>``` tag as  The ```<nav>``` [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents a section of a page whose purpose is to provide navigation links, either within the current document or to other documents. Common examples of navigation sections are menus, tables of contents, and indexes.